# The full name of the topic, including prefix /
string topic_name

# The amount of messages per second since the last statistic
uint8 traffic